### PR TITLE
Added a flow for setting up gdeploy in an imported cluster

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/commons#264
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@Tendrl/tendrl-core plz review

File: tendrl/commons/flows/setup_gdeploy/__init__.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                    NS.etcd_orm.client.read(
+                        "/queue/%s/status" % job_id
+                    ).value
+                )
+            if all(status == "finished" for status in all_status):
+                all_ssh_jobs_done = True        
+
+        return True
+
+    def load_definition(self):
```
Comment (r0h4n) [inline tendrl/commons/flows/setup_gdeploy/__init__.py]: You cannot override load_definition(),  either mark this flow with class variable "internal=True" or define this flow correctly in definitions (.yml) file

Comment (shtripat) [inline tendrl/commons/flows/setup_gdeploy/__init__.py]: Done

Comment (r0h4n) [inline tendrl/commons/flows/setup_gdeploy/__init__.py]: I dont see new commits, is this done?

**Pull request closed without merging**